index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+   <meta charset='utf-8'>
+   <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+   <title>My Portfolio</title>
+   <meta name='viewport' content='width=device-width, initial-scale=1'>
+   <link rel='stylesheet' type='text/css' media='screen' href='styles.css'>
+   <script src='main.js'></script>
+</head>
+<body>
+   <header>
+      <div class="logo">
+         <h3>My Logo</h3>
+      </div>
+      <div class="hamburger">
```

Comment:
- [ ] Kindly use a `nav` tag instead of a `div` tag. Using a `nav` tag is more semantic than using a `div` tag to wrap the _**hamburger**_ button. 👍